base/workflow/R/do_conversions.R

```diff
@@ -28,6 +28,12 @@ do_conversions <- function(settings, overwrite.met = FALSE, overwrite.fia = FALS
       next
     }
     
+  # Check for existing file paths , skips if the file exists
+    if (!is.null(input$path) && is.null(input$force)) {
+      PEcAn.logger::logger.info("Skipping input with existing path:", input.tag)
+      next
+    }
+    
     input.tag <- names(settings$run$input)[i]
     PEcAn.logger::logger.info("PROCESSING: ",input.tag)
```

Comment:
input.tag not yet defined at logger call time -- let's move it down
```suggestion
    input.tag <- names(settings$run$input)[i]
    PEcAn.logger::logger.info("PROCESSING: ",input.tag)

    # Check for existing file paths , skips if the file exists
    if (!is.null(input$path) && is.null(input$force)) {
      PEcAn.logger::logger.info("Skipping input with existing path:", input.tag)
      next
    }
```